bot/client.js

```diff
@@ -15,6 +15,16 @@ client.on('ready', () => {
   util.log('Bot Online and Ready', 0);
 });
 
+// TODO: Set Twitch notification
+
+// Message for new users
```

Comment:
Remove the old Welcome Feature, this should only include code for the live stream notification code.

Reply:
Thank you for this I am removing it now